backend/lcfs/db/models/compliance/ComplianceReport.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy.orm import relationship
+from lcfs.db.base import BaseModel, Auditable
+
+class ComplianceReport(BaseModel, Auditable):
+    __tablename__ = 'compliance_report'
+    __table_args__ = (
+        {'comment': "Main tracking table for all the sub-tables associated with a supplier's annual compliance report"}
+    )
+    
+    compliance_report_id = Column(Integer, primary_key=True, autoincrement=True, comment="Unique identifier for the compliance report")
```

Comment:
There is a type column in the compliance report index table in the UI, which has columns like "Quarter 1 report, Compliance Report, Supplementary report", do we need to take this into consideration?

Reply:
We're not going to tackle supplemental and quarterly reporting later. I think its better to just focus on the static page for now.